consensus/enclave/impl/build.rs

```diff
@@ -25,4 +26,41 @@ fn main() {
 
     fs::write(&env.out_dir().join("target_features.rs"), &contents)
```

Comment:
Could also benefit from a check if the contents haven't changed (since cargo relies on modification time)